charts/recommended.yaml

```diff
@@ -167,63 +166,149 @@ subjects:
 
 ---
 
+################################
+### Services & Ingresses
+################################
+
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    k8s-app: kubernetes-dashboard
+    role: web
+  name: dashboard-web
+  namespace: kubernetes-dashboard
+spec:
+  ports:
+    - name: http
+      port: 80
+      targetPort: 8000
+  selector:
+    k8s-app: kubernetes-dashboard
+    role: web
+
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    k8s-app: kubernetes-dashboard
+    role: api
+  name: dashboard-api
+  namespace: kubernetes-dashboard
+spec:
+  ports:
+    - name: api
+      port: 9000
+      targetPort: 9000
+  selector:
+    k8s-app: kubernetes-dashboard
+    role: api
+
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    k8s-app: metrics-scraper
+  name: metrics-scraper
+  namespace: kubernetes-dashboard
+spec:
+  ports:
+    - port: 8001
+      targetPort: 8001
+  selector:
+    k8s-app: metrics-scraper
+
+---
+
+kind: Ingress
+apiVersion: networking.k8s.io/v1
+metadata:
+  labels:
+    k8s-app: kubernetes-dashboard
+  annotations:
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
+    cert-manager.io/issuer: selfsigned
+  name: kubernetes-dashboard
+  namespace: kubernetes-dashboard
+spec:
+  ingressClassName: nginx
+  tls:
+    - hosts:
+        - kubernetes.docker.internal
+      secretName: kubernetes-dashboard-certs
+  rules:
+    - host: kubernetes.docker.internal
+      http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: dashboard-web
+                port:
+                  name: http
+          - path: /api/v1
```

Comment:
Hi @floreks 

Here, path should point to `/api` else `/api/sockjs` for _exec into pod_  does not work.